Sources/CShims/include/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module CShims {
```

Comment:
Using explicit module maps breaks some older Xcode functionality in fun ways, but we don't need it. If you change the name of the `shims.h` header to be exactly the same as the module name (in this case, I've proposed `CAsyncHTTPClient`, which would give you the header file `CAsyncHTTPClient.h`, then SwiftPM will autogenerate the module map as needed.